packages/backend/src/interface/request.ts

```diff
@@ -0,0 +1,9 @@
+export interface MessageQuery {
+  lastId?: string
+  limit?: string
+}
+
+export interface FanartQuery {
+  lastId?: string
+  limit?: string
+}
```

Comment:
Maybe create a `PaginateQuery` or something?

Reply:
good idea, will do that